src/main/kotlin/gg/essential/elementa/markdown/MarkdownComponent.kt

```diff
@@ -34,6 +35,7 @@ class MarkdownComponent(
     private val codeFontPointSize: Float = 10f,
     private val codeFontRenderer: FontProvider = ElementaFonts.JETBRAINS_MONO,
     private val disableSelection: Boolean = false,
+    val imageCache: ImageCache? = null
```

Comment:
And please add the trailing comma so the next person that wants to add an argument gets a nice single-line diff as well.
(same for the `@JvmOverloads` one)